train/training.py

```diff
@@ -93,6 +94,61 @@ def convert_labels_to_csv(data, tagging_output_file_path):
         print("An error occurred attempting to write to file at {0}:\n\n{1}".format(tagging_output_file_path,e))
         raise
 
+def upload_data_post_training(training_id):
+    tagging_output_file_path =  os.path.expanduser(config.get('tagged_output'))
+    function_url = config.get("url") + "/api/classification"
+    query = {
+        "userName": config.get('tagging_user')
+    }
+
+    # First, we need to get a mapping of class names to class ids
+    classes = ""
+    with open(tagging_output_file_path + "/tagged.csv") as f:
```

Comment:
@samiyaakhtar we should pass in the entire path as a param. No hardcoding of file names

Reply:
Done